scripts/run_tests.sh

```diff
@@ -46,13 +46,13 @@ if [ "$run_docker" == 1 ]; then
         --template \
         "/kb/dev_container/narrative/src/config.json.templ:/kb/dev_container/narrative/kbase-extension/static/kbase/config/config.json" \
         kbase-narrative --no-browser --port=$JUPYTER_PORT 2>&1 | tee $OUTPUT_FILE &
-    bg_pid=$!
+    bg_pid=$(jobs -p)
```

Comment:
Tests ongoing - I want to see how it behaves on GHA.
But when running on my local MacOS, `$bg_pid` had the pid of the `tee` process, not `kbase-narrative`. Running `jobs -p` fixes that.